components/dashboard/src/teams/TeamUsageBasedBilling.tsx

```diff
@@ -21,7 +21,7 @@ export default function TeamUsageBasedBilling() {
     const { teams } = useContext(TeamsContext);
     const location = useLocation();
     const team = getCurrentTeam(location, teams);
-    const { showUsageBasedUI } = useContext(PaymentContext);
+    const { showUsageBasedUI, currency } = useContext(PaymentContext);
```

Comment:
FIXME: This auto-detects the currency:
- EUR for European countries (but only in production)
- USD for all other countries (and always USD in preview envs)

But -- maybe we'll want to ask users to confirm this choice? (E.g. if the user is based in Europe, but signing up for a US-based team with a credit card in USD, then they'll likely want invoices in USD not EUR)

Reply:
Leaving this as a follow-up UX improvement.